packages/flutter/test/widgets/scrollbar_test.dart

```diff
@@ -2565,4 +2565,35 @@ void main() {
 
     // Go without throw.
   });
+
+  testWidgets('Track offset respects padding', (WidgetTester tester) async {
+    final ScrollController scrollController = ScrollController();
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: MediaQuery(
+          data: const MediaQueryData(
+            padding: EdgeInsets.all(50.0),
+          ),
+          child: RawScrollbar(
+            controller: scrollController,
+            minThumbLength: 21,
+            minOverscrollLength: 8,
+            isAlwaysShown: true,
```

Comment:
nit: Use thumbVisibility instea.

Reply:
Ah thank you! I came across this while working on another proposal and just copied another test in a rush. 🙃